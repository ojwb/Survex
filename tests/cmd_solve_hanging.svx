; pos=fail warn=0 error=
; Regression test distilled from example from Carlos Grohmann
*begin A
1	2	1.23 123 12
*end A
*solve
*begin B
3	4	2.34 234 23
*end B
