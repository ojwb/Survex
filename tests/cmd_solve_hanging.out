./cmd_solve_hanging.svx:4: info: Survey has no fixed points. Therefore I've fixed a.1 at (0,0,0)

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Removing trailing traverses...

Concatenating traverses...

Simplifying network...
cavern: error: Survey not all connected to fixed stations
The following survey stations are not attached to a fixed point:
./cmd_solve_hanging.svx:8: info: b.3
